packages/server/src/api/controllers/market-data.ts

```diff
@@ -74,22 +74,23 @@ async function getPoolIndicators(
     return { marketData, indicators: poolIndicators };
 }
 
+const cacheConfig = { maxAge: 15, sMaxAge: 300, public: true, mustRevalidate: true };
 export default Router()
     .get(
         '/daily',
         cacheMiddleware(300),
```

Comment:
we still cache on our end for 5 minutes since cache can be busted very easily by clients